tasks/test_results_finisher.py

```diff
@@ -357,6 +357,7 @@ def process_impl_within_lock(
                 "Detected failure on test that has been identified as flaky",
                 extra=dict(
                     success=success,
+                    notifier_result=notifier_result.value,
```

Comment:
Would using the `.name` be more readable?

Reply:
the definition of the notifier result looks like: 
```
class NotifierResult(Enum):
    COMMENT_POSTED = "comment_posted"
    TORNGIT_ERROR = "torngit_error"
    NO_PULL = "no_pull"
```
so i think either name or value should be fine